### DISCUSSION TITLE
warning: ghost idx out of upper bound

### DISCUSSION THREAD

**Discussion by feng-phoenix-wang** [deepmodeling/deepmd-kit#27] [Q&A]
I have already wrapped the coordination and increased sel_a to a very large number. But there are always this warnings. How could solve this type of warnings?
`{`
`    "_comment": " model parameters",`
`    "use_smooth": false,`
`    "sel_a": [ 190, 190, 190],`
`     "sel_r": [ 190, 190, 190 ],`
`    "rcut_a": -1,`
`   "rcut": 6.2,`
`    "axis_rule": [`
`        0, 1, 0,`
`        0, 1, 1,`
`        0, 0, 0,`
`        0, 1, 0,`
`        0, 0, 0,`
`        0, 0, 1`
`   ],`
`    "fitting_neuron": [`
`       240,`
`        80,`
`        20,`
`        10`
`    ],`
`    "systems": [`
`        "../data/data/"`
`    ],`
`    "set_prefix": "set",`
`    "stop_batch": 1000000,`
`    "batch_size": 8,`
`    "start_lr": 0.001,`
`    "decay_steps": 4000,`
`    "decay_rate": 0.95,`
`    "start_pref_e": 0.02,`
`    "limit_pref_e": 12,`
`    "start_pref_f": 1000,`
`    "limit_pref_f": 1,`
`    "start_pref_v": 0,`
`    "limit_pref_v": 0,`
`    "num_threads": 4,`
`    "seed": 1,`
`    "disp_file": "lcurve.out",`
`    "disp_freq": 100,`
`    "numb_test": 100,`
`    "save_freq": 2000,`
`    "save_ckpt": "model.ckpt",`
`    "restart": false,`
`    "load_ckpt": "model.ckpt",`
`    "disp_training": true,`
`    "time_training": true`
`}`

**Comment by hc317** [comment]
Have you solved this problem? I had the same problem when I called the dp_test script. I found that the atom type is inconsistent with the type.raw file. I didn't see the same warnings after the correction. I hope it can help you.
